nifi-nar-bundles/nifi-kudu-bundle/nifi-kudu-processors/src/main/java/org/apache/nifi/processors/kudu/PutKudu.java

```diff
@@ -213,58 +198,18 @@
         return rels;
     }
 
+    protected KerberosUser kerberosUser;
+    protected KuduSession kuduSession;
+    protected String tableName;
 
     @OnScheduled
     public void onScheduled(final ProcessContext context) throws IOException, LoginException {
-        final String kuduMasters = context.getProperty(KUDU_MASTERS).evaluateAttributeExpressions().getValue();
-        operationType = OperationType.valueOf(context.getProperty(INSERT_OPERATION).getValue());
+        tableName = context.getProperty(TABLE_NAME).evaluateAttributeExpressions().getValue();
```

Comment:
This is not necessary as we are evaluating the table name L257 based on FFs attributes.